src/main/java/org/apache/ibatis/parsing/XNode.java

```diff
@@ -130,71 +130,47 @@ public String getStringBody() {
   }
 
   public String getStringBody(String def) {
-    if (body == null) {
-      return def;
-    } else {
-      return body;
-    }
+    return body == null ? def : body;
   }
 
   public Boolean getBooleanBody() {
     return getBooleanBody(null);
   }
 
   public Boolean getBooleanBody(Boolean def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Boolean.valueOf(body);
-    }
+    return body == null ? def : Boolean.valueOf(body);
   }
 
   public Integer getIntBody() {
     return getIntBody(null);
   }
 
   public Integer getIntBody(Integer def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Integer.parseInt(body);
-    }
+    return body == null ? def : Integer.valueOf(Integer.parseInt(body));
   }
 
   public Long getLongBody() {
     return getLongBody(null);
   }
 
   public Long getLongBody(Long def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Long.parseLong(body);
-    }
+    return body == null ? def : Long.valueOf(Long.parseLong(body));
```

Comment:
@nothingzhl Well maybe not these as being ok.  Please after your rebase, just use the smarter if without unnecessary wrapping at least in this initial PR.  That way it's just one change at a time (ie smart if style / but without changing the overall syntax).